src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/3.sql

```diff
@@ -1,344 +1,538 @@
 -- Style guide: please see: https://github.com/ktaranov/sqlserver-kit/blob/master/SQL%20Server%20Name%20Convention%20and%20T-SQL%20Programming%20Style.md
+
+/*************************************************************
+Full text catalog creation
+**************************************************************/
+IF NOT EXISTS (
```

Comment:
These changes are not needed. It is already re-runnable and more readable and cleaner. We should follow similar model in 4.sql

Reply:
Ah, my mistaken! I'll abandoned this one then